OWNERS

```diff
@@ -1,17 +1,8 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
-  - 4n4nd
-  - HumairAK
-  - tumido
   - durandom
   - goern
 
 reviewers:
-  - 4n4nd
-  - anishasthana
   - harshad16
```

Comment:
```suggestion
reviewers: {}
```

shall we clear out reviewers ,as we have deprecated the most of the bits.
having a reviewers might not help out.